tests/unit/test_business_logic.py

```diff
@@ -601,26 +615,35 @@ def test_provider_request_release_files(mock_old_api, mock_notifications, bll, f
     }
 
     old_api.create_release.assert_called_once_with(
-        "workspace", "request_id", json.dumps(expected_json), checker.username
+        "workspace", "request_id", json.dumps(expected_json), checkers[0].username
     )
     old_api.upload_file.assert_called_once_with(
-        "jobserver_id", relpath, abspath, checker.username
+        "jobserver_id", relpath, abspath, checkers[0].username
     )
 
     notification_responses = parse_notification_responses(mock_notifications)
     # Notifications expected for:
     # - set status to submitted
     # - set status to partially reviewed
     # - set status to reviewed
+    # - set status to returned
+    # - set status to submitted
+    # - set status to partially reviewed
+    # - set status to reviewed
+    # - set status to returned
     # - set status to approved
     # - set status to released
```

Comment:
do we really need this comment? `expected_notifications` looks quite self-explanatory?